common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -122,17 +122,16 @@ iframe[seamless]{
 }
 
 div.problem-progress {
-  @include padding-left($baseline/4);
-  @extend %t-ultralight;
   display: inline-block;
   color: $gray-d1;
-  font-weight: 100;
-  font-size: em(16);
+  font-size: em(14);
 }
 
 // +Problem - Base
 // ====================
 div.problem {
+  padding: ($baseline) 0 0 0 ;
```

Comment:
@alisan617 stylistically, is it better to just specify the padding-top instead of all 4?